lib/rubycritic/platforms/linux.rb

```diff
@@ -4,14 +4,10 @@ module Rubycritic
   module Platforms
     class Linux < Base
       BROWSER_MAP = {}.tap do |hash|
-        hash[:chrome] = 'google-chrome'
+        hash[:chrome]   = 'google-chrome'
         hash[:chromium] = 'chromium-browser'
-        hash[:firefox] = 'firefox'
-        hash.default = 'firefox'
-      end
-
-      def can_open?
-        platform == :linux || platform == :unix
+        hash[:firefox]  = 'firefox'
+        hash.default    = 'firefox'
```

Comment:
I think `hash.default` could be the default browser in the current system rather than this hardcoded default. I my case, default is google chrome.